### DISCUSSION TITLE
Incorrect legend in logout flow

### DISCUSSION THREAD

**Discussion by saki-is-username** [ory/hydra#2981] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43903579/153329655-d6ee34a8-ed7f-4870-9457-bc562582267d.png)
![image](https://user-images.githubusercontent.com/43903579/153329785-041d5993-cfe3-4061-813b-ad5147691a67.png)
![image](https://user-images.githubusercontent.com/43903579/153330397-6efd68e3-55f2-4ed9-9457-3787a30ae5a0.png)
While I was using Hydra, situations which were not found in the diagram occurred. I signed in successfully with postman, cleared the cookie of it, and log out with id_token_hint, state, and post_logout_redirect_URI(Rememer Login is not selected). According to the diagram, the URL is supposed to jump to logout_verifier page after the logout request is sent. However, the page goes to post_logout_url. 
My question is how to explain the situation? Is there any inconsistent part in the diagram which needs to modify?

**Comment by vinckr** [comment, marked as answer]
Hello @saki-is-username 
Apologies for the late answer.
Would you be able to provide some steps for me to reproduce this using a browser and the [quickstart](https://www.ory.sh/docs/hydra/5min-tutorial)?

Otherwise try the same flow with cUrl, this might also be related to postman being a browser (electron) by itself.
